core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -547,6 +547,9 @@
    */
   VALUES,
 
+  /** {@code FIRST_LAST_VALUE} expression. */
+  FIRST_LAST_VALUE,
```

Comment:
I am not sure if it is necessary for this expression to have its own SqlKind, would it be in either SqlKind.OTHER or SqlKind.OTHER_FUNCTION? Could you provide more context on why FIRST_LAST_VALUE should have it's own SqlKind?

Reply:
Having a specific SqlKind may be preferable by downstream consumers. What do you think @dasch-1 ?

Reply:
Yes, having a distinct SqlKind is always preferred by downstream consumers.